schema-derivatives-options.graphql

```diff
@@ -1015,6 +1030,12 @@ type LiquidityPoolHourlySnapshot @entity(immutable: true) {
   " Hourly Deposit trade notional volume occurred in this pool for a specific input token, in USD. The ordering should be the same as the pool's `inputTokens` field. "
   hourlyDepositVolumeByTokenUSD: [BigDecimal!]!
 
+  " All Inflow trade notional volume occurred in this pool for a specific input token, in native amount. The ordering should be the same as the pool's `inputTokens` field. "
+  _cumulativeDepositVolumeByTokenAmount: [BigInt!]!
```

Comment:
Why do we want to add this as a helper field? If we add this here there should be a counterpart in `LiquidityPool`

Reply:
Fixed at 5b75305b341ea53c57e08fee4dab7eb7420db7e9 I may add them by mistake and they don't need in the schema template.